Add Beszel to Monitoring (#606)
awesome-foss/awesome-sysadmin: README.md

- [Adagios](http://adagios.org/) - Web based Nagios interface for configuration and monitoring (replacement to the standard interface), and a REST interface. ([Source Code](https://github.com/opinkerfi/adagios)) `AGPL-3.0` `Docker/Python`
- [Alerta](https://alerta.io/) - Distributed, scalable and flexible monitoring system. ([Source Code](https://github.com/alerta/alerta)) `Apache-2.0` `Python`

> - [Beszel](https://github.com/henrygd/beszel) - Beszel is a lightweight server monitoring platform that includes Docker statistics, historical data, and alert functions. `MIT` `Go`

Reviewer: ```suggestion
- [Beszel](https://beszel.dev/) - Lightweight server monitoring platform that includes Docker statistics, historical data, and alert functions. ([Source Code](https://github.com/henrygd/beszel)) `MIT` `Go`
```
